batch/src/main/java/com/hartwig/batch/BatchArguments.java

```diff
@@ -45,7 +47,7 @@ static BatchArguments from(String[] args) {
                     .outputBucket(commandLine.getOptionValue(OUTPUT_BUCKET))
                     .cmek(commandLine.getOptionValue(CMEK, CommonArguments.DEFAULT_DEVELOPMENT_CMEK))
                     .network(commandLine.getOptionValue(PRIVATE_NETWORK, DEFAULT_NETWORK))
-                    .usePublicImage(false)
+                    .imageName(commandLine.getOptionValue(IMAGE_NAME, EMPTY))
```

Comment:
I think better to just not expose the image name argument to batch, then you can also remove the check for empty string in the compute engine.